packages/ui/src/components/wui-text/index.ts

```diff
@@ -0,0 +1,63 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import { classMap } from 'lit/directives/class-map.js'
+
+import { globalStyles } from '../../utils/ThemeUtil'
+import styles from './styles'
+
+type Variant =
+  | 'lg-medium'
+  | 'lg-semibold'
+  | 'md-bold'
+  | 'md-medium'
+  | 'md-numerals'
+  | 'md-semibold'
+  | 'sm-medium'
+  | 'sm-semibold'
+  | 'xxs-bold'
+
+type Color = 'accent' | 'error' | 'inverse' | 'primary' | 'secondary' | 'tertiary' | 'success'
+
+@customElement('wui-text')
+export class WuiText extends LitElement {
+  public static styles = [globalStyles, styles]
```

Comment:
Something to research in `lit`, do you think we can find a better solution to global styles i.e. instead of defining them in each component, maybe we can define one time somewhere?